cmd/plant/gen.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"automatica.team/plant"
+	"automatica.team/plant/cmd/plant/do"
+	"automatica.team/plant/cmd/plant/exec"
+	"fmt"
+	"github.com/spf13/cobra"
+	"path/filepath"
+)
```

Comment:
GoLand -> Settings -> (in Search) gofmt -> Sorting type

<img width="984" alt="image" src="https://github.com/user-attachments/assets/42e62490-80ef-4dc8-990e-71ae564967dd">